modules/ROOT/partials/configuration/valid_elements.adoc

```diff
@@ -85,6 +88,7 @@ tinymce.init({
 
 This example string tells {productname} to remove all elements that are not a `'a, strong, div or br'` element, convert `'b'` elements to `'strong'` elements, default '[.code]``target``' to '[.code]``_blank``' and keep the '[.code]``href``', '[.code]``target``' and '[.code]``align``' attributes of the elements.
```

Comment:
There's some conversion issues here, as the single quotes are outside the code blocks in about 50% of the cases.

Reply:
Nope, that’s how it is [in the original](https://www.tiny.cloud/docs/configure/content-filtering/#exampleusingvalid_elements). Will fix it though

Reply:
I think instead of current

```
This example string tells {productname} to remove all elements that are not a `'a, strong, div or br'` element, convert `'b'` elements to `'strong'` elements, default '[.code]``target``' to '[.code]``_blank``' and keep the '[.code]``href``', '[.code]``target``' and '[.code]``align``' attributes of the elements.
```

it just should be like this:

```
This example string tells {productname} to remove all elements that are not a `a`, `strong`, `div` or `br` element, convert `b` elements to `strong` elements, default `target` to `_blank` and keep the `href`, `target` and `align` attributes of the elements.
```

Reply:
If it's like this in the original then let's just leave it as is for now. This again could be one of those controversial decisions that we really need to pull in say Brian...